editoast/src/client/mod.rs

```diff
@@ -68,6 +69,16 @@ pub struct ImportTrainArgs {
     pub path: PathBuf,
 }
 
+#[derive(Args, Debug, Derivative)]
+#[derivative(Default)]
+#[command(about, long_about = "Export a train given a timetable id")]
```

Comment:
I missed it in the import PR but all `long_about` and `help` arg can be defined using `///` rust documentation. This way it's both documented and used by clap. We could do this for all commands in another PR for more consistency.

Reply:
Noted, I will open an issue and do this on another PR, I'm going to keep this one this way in order to stay consistent with current code convention, wdyt ?